src/material/slider/slider-input.ts

```diff
@@ -386,6 +386,10 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
     this._updateWidthActive();
     this._slider._updateDimensions();
 
+    if (this._platform.IOS) {
```

Comment:
Capturing what we talked about: we should have a comment here explaining why we need the special case.

Reply:
I realized there was another if statement above this that already has a comment describing this special case. I moved this early exit into there so the special casing is all in one place & documented together